feature/google/src/bc5SupportDev/java/com/revenuecat/purchases/google/storeProductConversions.kt

```diff
@@ -56,3 +58,53 @@ fun ProductDetails.toStoreProduct(offerToken: String, pricingPhases: ProductDeta
         offerToken,
         pricingPhases
     )
+
+fun ProductDetails.toStoreProduct(rcPackage: Package): StoreProduct? {
+    val duration = rcPackage.duration ?: return null // TODO error here, shoudln't be trying to make productdetails if BC5 not enabled
+    val offers = listOf()
+
+    subscriptionOfferDetails?.forEach { offerDetails ->
```

Comment:
this was my idea for nesting the offers inside the storeproduct. as i was going through it, i think it might make sense to create a new `Offer` object. having base plans and offers be the same object is confusing -- it took me a while to understand that from the billingclient, and ideally we can keep our customers from that confusion